src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -134,10 +134,23 @@ public function getNode($path)
 
         $cacheKey = "nodes: $path, ".$this->workspaceName;
         if (isset($this->caches['nodes']) && (false !== ($result = $this->caches['nodes']->fetch($cacheKey)))) {
+            if ('' === $result) {
+                throw new ItemNotFoundException("Item $path not found in workspace ".$this->workspaceName);
```

Comment:
Would be nice to use `sprintf` here and quote the values, `sprintf('Item "%s" not found in workspace "%s"' ..`